### PR TITLE
Owlot/dw 605/flyarrowio website in

### PR DESCRIPTION
**Pull request by owlot**
Current status can be found on the demo site: https://www.dmo.flyarrow.io/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1005 @@
...
+                        <img class="card-img" src="/assets/img/Icon-vtolcargo.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title"> CARGO <span>flights</span></h3>
+                        <p>On demand cargo flights both private to your own users as dispatch-able flights for others. Deploy your aircraft on our network and start shipping. </p>
```
Comment (GoodluckH) [inline index.html:99]: I think this is pitching to shippers. For operators who own aircraft, we can pitch to them in the Asset Manager card. And I think these product cards should be clickable so as to direct users to the product page. 

I would reword this to:
```
Take your middle-mile logistics to the next level. Cargo deliveries are now measured by minutes. Join today to revolutionize the way you transport goods.
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1005 @@
...
+        </div>
+    </div>
+
+    <div class="container-fluid py-5 about bg-primary text-light">
+        <div class="container">
+            <div class="row d-flex align-items-center">
+                <div class="col-3 p-4"> <img class="rounded-circle img-teamlead w-100" src="/assets/img/photo-dummy.jpg" alt="FlyArrow team lead"></div>
+                <div class="col-9 ">
+                    <h3>"We are here to change the future of air mobility"</h3>
+                    <p class="special1">- AM Smith, team lead of FlyArrow.io</p>
```
Comment (ServiceDog) [inline index.html:655]: "VTOL cargo and passenger networks are the next economic gateway. We'll be able to forge new connections to rural communities without the financial and environmental costs of traditional ground and fixed-wing infrastructure. The impact this will have on commuting, supply chains, and humanitarian response can't be overstated."

A.M. Smith, software and systems engineer for FlyArrow.io

**Review by ServiceDog — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1005 @@
...
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-fluid bg-main text-light">
+        <div class="row mb-5">
+            <div class="col-6 pe-5">
+                <h1>Building VTOL software for automated scheduled flights</h1>
+                <p>With our full-stack flightplan system you can take the pain out of launching a safe and fast system to organize and monetize on your VTOL assets and services.</p>
```
Comment (GoodluckH) [inline index.html:75]: This can be more prominent than just a <p> tag.

**Review by GoodluckH — CHANGES_REQUESTED** [review]
More to come. Seems like there are conflicts with @ServiceDog's comments. Let's resolve them first.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1005 @@
...
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-padlock.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Assets
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">Admin <span>Manager</span></h3>
```
[the remark this answers is not in the document]
Comment (ServiceDog) [reply index.html:130]: Would be 'Hangar', not a bad name

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1005 @@
...
+                    <button type="button" class="btn btn-info">Request a meeting</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <header class="container-fluid message">
+        <div class="row">
+            <div class="col-12 text-center">
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:66]: How about:
`FlyArrow.io is the innovative software development project of ARROW - a DAO committed to advancing planet-friendly air mobility as a fundamental human right. Discover the boundless potential of a better future, one flight at a time.`

The 'non-profit' part was mainly added because not everyone knows what a DAO is, but they can form an idea about a 'non-profit' organization. I'm not too fuzzed about using the word, as Arrow DAO -is- non-profit.

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:75]: `Unlock the potential of your VTOL assets with FlyArrow.io's community-grown vertiport network. With our dynamic scheduling, automated air traffic control, and digital compliance mechanisms, you can easily connect urban and rural populations and simplify the process of managing your assets.`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1005 @@
...
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-vtolcargo.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title"> CARGO <span>flights</span></h3>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:98]: Will change to `Cargo <span>flights</span>` for now, as the design expects a sub-title

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1005 @@
...
+                        <p>On demand cargo flights both private to your own users as dispatch-able flights for others. Deploy your aircraft on our network and start shipping. </p>
+                    </div>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:100]: Combining the two proposed sentences:
`Revolutionize your middle-mile logistics with FlyArrow.io - easily schedule on-demand deliveries to public and private vertiports, register and manage cargo aircraft, and enjoy cargo deliveries measured in minutes with automated job assignments and scheduling.`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1005 @@
...
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-vtolperson.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">Passenger <span>flights</span></h3>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:114]: Will change to `Rideshare <span>network</span>` for now, as the design expects a sub-title

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1005 @@
...
+                        <p>Mid-range VTOL air travel will be the future and it is closer than you think. Venture in as early adapter on this future and planet proof transportation that will cost you no software development time.</p>
+                    </div>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:116]: Combined the proposed changes into:
`Join the next great leap in human mobility with FlyArrow.io's vertiport network - an economic gateway connecting urban and rural populations via air taxis for rapid and reliable transportation. Be an early adopter and book your air taxi with FlyArrow.io.`

File: index.html [hunk 10]
```diff
@@ -0,0 +1,1005 @@
...
+                        <img class="card-img" src="/assets/img/Icon-padlock.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Assets
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">Admin <span>Manager</span></h3>
+                        <p>With all your ambitions, you will have to be able to manage your assets. Rather those are aircraft or vertiports, pilots, user rights and regulations, we've got you covered. Save time with our easy to use admin manager.</p>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html:131]: Combined into:
`Easily manage your assets with FlyArrow.io's one-stop-shop. Register, group, delegate, and maintain with ease. Analyze asset performance and manage vertipads, aircraft, and pricing models with our simple user interface.`

File: index.html [hunk 11]
```diff
@@ -0,0 +1,1005 @@
...
+            <div class="col-12 text-center">
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-fluid bg-main text-light">
+        <div class="row mb-5">
+            <div class="col-6 pe-5">
+                <h1>Building VTOL software for automated scheduled flights</h1>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html [hunk 12]
```diff
@@ -0,0 +1,1005 @@
+<!DOCTYPE html>
+<html lang="utf-8">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>VTOL autonomous flying software - Fly Arrow.io</title>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html [hunk 13]
```diff
@@ -0,0 +1,1005 @@
...
+                        <img class="card-img" src="/assets/img/Icon-vtolperson.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">Passenger <span>flights</span></h3>
+                        <p>Mid-range VTOL air travel will be the future and it is closer than you think. Venture in as early adapter on this future and planet proof transportation that will cost you no software development time.</p>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: reworded: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: reworded: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 2 above)
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: reworded: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: reworded: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html [hunk 14]
```diff
@@ -0,0 +1,1005 @@
...
+
+    <div class="container pb-5 flight-types">
+        <div class="row row-cols-1 row-cols-md-3 g-3">
+            <div class="col">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-vtolcargo.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html [hunk 15]
```diff
@@ -0,0 +1,1005 @@
...
+                </div>
+            </div>
+            <div class="col">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-vtolperson.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
```
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (owlot) [reply index.html]: reworded: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

File: index.html (hunk 3 above)
Comment (owlot) [reply index.html]: fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b

**Review by GoodluckH — APPROVED** [review]
